Fixes Issue #3512

jan-wi opened the pull request:
allow http address for local gitlab instance

<!--
Hi there - a lot of love for starting a Pull Request 😍. Please ensure the following things before creation - thank you!

- Create your Pull Request against the develop branch. We don't accept changes to stable branches.
- Add a reference to the issue you are trying to fix. Just add a # and the number.
- Don't run `bundle update`. It's a honorable thought of you but some dependency versions (e.g. pg) are broken 😢 Just use `bundle install` if you introduce new dependencies instead.
- Run `rubocop` and `coffeelint` on your changes to respect the code style guide.
- Add tests for your changes. Feel free to ask for support. We are happy to help you.

* The upper textblock will be removed automatically when you submit your Pull Request *
-->
Fixes Issue #3512

jan-wi committed:
Fixes Issue #3512

allow http address for local gitlab instance

thorsteneckel commented:
Thanks @jan-wi for your contribution! We should also do this change for GitHub because people might use their GitHub Enterprise installation with HTTP only as well? See: https://github.com/zammad/zammad/blob/3baec9c6060da269b0ede3126eef5fe987b1363c/lib/github/linked_issue.rb#L100

@mantas would  you mind taking this one over?

jan-wi committed:
Fixes Issue #3512 for github

Adapting change from 40533b64b39d3725750d31dd9f771db5c6a692ed from gitlab to github

jan-wi commented:
> We should also do this change for GitHub because people might use their GitHub Enterprise installation with HTTP only as well?

@thorsteneckel Change was adapted to the github lib as well ( 05ab1dae6e1fbef339c3e4c979ed447996dd9cee )

thorsteneckel commented:
Fixed with https://github.com/zammad/zammad/commit/a5de91064a2cf05855b0158ebece5e7819e3b840 - backport to 4.0.1 in the making 🚀
